campaignion_donation_amount/campaignion_donation_amount.variable.inc

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Information about configuration variables defined by this module.
+ */
+
+/**
+ * Implements hook_variable_info().
+ */
+function campaignion_donation_amount_variable_info($options) {
+  $v['campaignion_donation_amount_currency_symbol'] = [
+    'title' => t('Currency symbol'),
+    'description' => t('Currency symbol used for donation amounts.'),
+    'type' => 'string',
+    'default' => '€',
+    'localize' => FALSE,
```

Comment:
I agree that a currency symbol shouldn’t be translated, however practical experience shows it is. There is no other way to use different currency symbols for different languages that represent forms for different countries that use different currencies…

Reply:
I made it translatable.